### ISSUE TITLE
inject_sitepy fails if you have not created .pyc for site.py

### ISSUE THREAD

**Issue by paulsaccount** [toumorokoshi/uranium#2]
### Environment
uranium Version: 0.0.64
Platform: Mac OSX 10.10.2
Python Version: Python 3.4.2

Affected lines:
https://github.com/toumorokoshi/uranium/blob/ab8fede5db08cd355589b44a798f4ca320794ad5/uranium/virtualenv_manager.py#L72

### Repo steps
1. Create a fresh install of python3 (where no .pyc files exist)
2. Setup project with uranium
3. run uranium

### Expected result
uranium finishes without exception

### Actual result
Exception occurs. It looks like the file expects a .pyc to exist when parsing.
```
[21:39:30] ================
[21:39:30] STARTING URANIUM
[21:39:30] ================
[21:39:30] installing eggs...
[21:39:30] Adding develop-egg /Users/paulpavlish/Dropbox/behonest/code/behonest-api...
[21:39:30] Adding requirement httpretty...
[21:39:30] Adding requirement mock...
[21:39:30] Adding requirement coverage...
[21:39:30] Adding requirement nose...
[21:39:30] Adding requirement boto...
[21:39:30] Adding requirement docopt...
[21:39:30] Adding requirement flask...
[21:39:30] Adding requirement gunicorn...
[21:39:30] Adding requirement itsdangerous...
[21:39:30] Adding requirement psycopg2...
[21:39:30] Adding requirement pyyaml...
[21:39:30] Adding requirement requests...
[21:39:30] Adding requirement sqlalchemy...
[21:39:30] Adding develop-egg /Users/paulpavlish/Dropbox/behonest/code/behonest-api...
[21:39:32] Adding requirement Werkzeug>=0.7...
[21:39:32] Adding requirement Jinja2>=2.4...
[21:39:34] Adding requirement markupsafe...
[21:39:34] running part test...
[21:39:34] Adding requirement uranium.plugin.shell...
[21:39:39] ================
[21:39:39] URANIUM FINISHED
[21:39:39] ================
Traceback (most recent call last):
  File "/Users/paulpavlish/Dropbox/behonest/code/behonest-api/.uranium/bin/uranium", line 9, in <module>
    load_entry_point('uranium==0.0.64', 'console_scripts', 'uranium')()
  File "/Users/paulpavlish/Dropbox/behonest/code/behonest-api/.uranium/lib/python3.4/site-packages/uranium/__init__.py", line 47, in main
    _inject_uranium_into_venv(uranium_dir, uranium)
  File "/Users/paulpavlish/Dropbox/behonest/code/behonest-api/.uranium/lib/python3.4/site-packages/uranium/__init__.py", line 75, in _inject_uranium_into_venv
    inject_sitepy(uranium_dir)
  File "/Users/paulpavlish/Dropbox/behonest/code/behonest-api/.uranium/lib/python3.4/site-packages/uranium/virtualenv_manager.py", line 63, in inject_sitepy
    inject_into_file(site_py_file, SITE_PY_INJECTION)
  File "/Users/paulpavlish/Dropbox/behonest/code/behonest-api/.uranium/lib/python3.4/site-packages/uranium/virtualenv_manager.py", line 77, in inject_into_file
    with open(path) as fh:
FileNotFoundError: [Errno 2] No such file or directory: b'/Users/paulpavlish/Dropbox/behonest/code/behonest-api/bin/../lib/python3.4/site.p'
```

**Comment by toumorokoshi** [comment]
@paulsaccount this should be fixed as of uranium 0.66. Can you verify this?

**Comment by paulsaccount** [comment]
Yay! I am not getting this issue anymore, thanks for resolving so quickly.
